src/server/simple_wml.hpp

```diff
@@ -37,7 +37,7 @@ class string_span
 	{}
 	string_span(const char* str, int size) : str_(str), size_(size)
 	{}
-	string_span(const char* str) : str_(str), size_(strlen(str))
+	explicit string_span(const char* str) : str_(str), size_(strlen(str))
```

Comment:
Drop this commit. There is nothing wrong with implicitly converting a `const char*` to a `string_span`.